src/admin/class-admin.php

```diff
@@ -21,6 +20,26 @@
  */
 class Admin implements Service {
 
+  /**
+   * Manifest Data Interface.
+   *
+   * @var Manifest_Data
+   *
+   * @since 4.0.0 Init.
+   */
+  protected $manifest;
+
+  /**
+   * Contruct method
```

Comment:
Typo, plus `__construct` docblock seems a bit, unnecessary written this way. Maybe something along the lines would be better:

```php
/**
 * Create a new admin instance that injects manifest data
 * for use in assets registration.
```

Reply:
👍